server/src/logic.js

```diff
@@ -578,6 +676,31 @@ export const getHandlers = ({ models, creators: Creators, push, pubsub }) => {
           order: [['createdAt', 'DESC']],
         });
       },
+      setEventNotifications(args, ctx) {
+        console.log(args);
+        const { eventId, enabled } = args.notifications;
+        getAuthenticatedUser(ctx).then(user =>
```

Comment:
I dont see how this can work.. you're not returning the promise.

Reply:
I dont need to react to the outcome, I just need it to happen.

Reply:
this one you definitely need to fix - the error produced on L685 will never reach the client, otherwise.